### DISCUSSION TITLE
What's The skiovox exploit for v124?

### DISCUSSION THREAD

**Discussion by Balistic123** [3kh0/ext-remover#1137] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
What's The skiovox exploit for v124? I was able to downgrade can someone give  me the tutorial? also what can I do with skiovox? does it effect my computer in anyway?

**Comment by Balistic123** [comment]
@AshtonDavies

**Comment by AshtonDavies** [comment]
It doesn't mess up your Chromebook in any serious way. It does store data in a kiosk profile, but it's pretty much harmless.

I can give the tutorial if you really want. It might not work for you, I forgot how it's done.

**Reply by Balistic123** [reply]
ok thanks

**Reply by Balistic123** [reply]
and yes please I would like the tutorial :)

**Reply by Balistic123** [reply]
@AshtonDavies also what version of v124 should I downgrade to? and yes please I would like the tutorial :)

**Reply by Balistic123** [reply]
@AshtonDavies

**Reply by AshtonDavies** [reply, marked as answer]
@Balistic123 Here is the SKIOVOX on v124 kajig (credit to **TeaEyeEighty4** on Discord):

# Skiovox on v124

prereqs:
- don't be stupid and have fast fingers
- sign in as existing user on network not available page

**Steps:**
- Sign out and turn off your wifi
- Open kiosk app of choice and wait for the network page to show
***DO NEXT STEPS AS FAST AS POSSIBLE***
- Click add network and quickly type in random letters and hit enter
- Quickly press diagnose, wifi, open in settings, and then press Close at the bottom right to exit out back to the wifi page
- As fast as you can press log in as existing and sign in
- Close the school window in the top right and there should be an unblocked page behind it

if all went well you should have an unblocked skiovox container.

**currently unable to use breakout since the load unpacked dialog won't show and new tab redirects to kiosk (probably having a braindead moment)**

**Reply by AshtonDavies** [reply]
Hopefully I got the right one...

**Reply by Balistic123** [reply]
so you cant install skiovox helper because the load unpacked dialog doesn't work? also no skebstore or anything? also what does skiovox breakout do

**Reply by Balistic123** [reply]
> @Balistic123 Here is the SKIOVOX on v124 kajig (credit to **TeaEyeEighty4** on Discord):
> 
> # Skiovox on v124
> prereqs:
> 
> * don't be stupid and have fast fingers
> * sign in as existing user on network not available page
> 
> **Steps:**
> 
> * Sign out and turn off your wifi
> * Open kiosk app of choice and wait for the network page to show
>   _**DO NEXT STEPS AS FAST AS POSSIBLE**_
> * Click add network and quickly type in random letters and hit enter
> * Quickly press diagnose, wifi, open in settings, and then press Close at the bottom right to exit out back to the wifi page
> * As fast as you can press log in as existing and sign in
> * Close the school window in the top right and there should be an unblocked page behind it
> 
> if all went well you should have an unblocked skiovox container.
> 
> **currently unable to use breakout since the load unpacked dialog won't show and new tab redirects to kiosk (probably having a braindead moment)**

@AshtonDavies Two things, The load unpacked button doesn’t work so that means no Skiovox helper? And I need to have a sign in as existing user to do this exploit?

**Reply by AshtonDavies** [reply]
Yes, the load unpacked button is unavailable, which was a note in the kajig.

I think the sign in as existing user option should be available if done correctly.

There is probably a dozen or so other methods out there. I don't bother one bit to keep track of them all, but maybe some of them would work.

**Reply by Balistic123** [reply]
Do you know where I could find them like a discord or something because currently I have no idea where to start @AshtonDavies

**Reply by AshtonDavies** [reply]
> Do you know where I could find them like a discord or something because currently I have no idea where to start @AshtonDavies 

[Titanium Network](https://discord.gg/unblock) is where you should be looking instead of here.

You will need to verify to access the [kajigs](https://discord.com/channels/419123358698045453/1019643248609865728) channel.

**Reply by AshtonDavies** [reply]
[This server](https://discord.gg/FDyT78xUps) may also have support for SKIOVOX if you want.

**Reply by Balistic123** [reply]
can you explain how skiovox works and how the exploit actually causes skiovox? @AshtonDavies so I can understand it better hoping to find one for v126

**Reply by AshtonDavies** [reply]
> can you explain how skiovox works and how the exploit actually causes skiovox? @AshtonDavies so I can understand it better hoping to find one for v126

Read the [PDF](https://skiovox.com) written by Bypassi.

Or his [legacy writeup](https://skiovox.glitch.me).

**Comment by Balistic123** [comment]
@AshtonDavies so you cant install skiovox helper because the load unpacked dialog doesn't work? also no skebstore or anything? also what does skiovox breakout do

**Reply by AshtonDavies** [reply]
> also what does skiovox breakout do

SKIOVOX Breakout is for running scripts.

**Reply by Balistic123** [reply]
@AshtonDavies could you provide a video tutorial of this exploit? Skiovox v124 re-mastered REQUIREMENTS - Needs to be v124 or higher

Sign out
Turn off your internet
Click add person
Hold the esc key
Click apps
Click on any kiosk app
Click add network
Put random letters in and click enter
Click diagnose
Click wifi
Click open in settings
Close out the diagnose widow (click the X at the top right)
Click sign in as existing user (it should show up now)
Sign in
Close out the school window (or double click the top and click the window behind it)
Profit. If your school policy blocked google, use a proxy like interstellar to browse the internet.

**Reply by AshtonDavies** [reply]
No, I am not providing a video tutorial.

**Reply by synapt0** [reply]
> Multiple user sign-in or something was disabled for me. How do I get around that if you don't mind me asking?

I cannot press escape to get out of it is what I meant.

**Reply by SpaceMonke135** [reply]
I don't think there is anyway around it.

**Reply by SpaceMonke135** [reply]
I tried that before too

**Reply by synapt0** [reply]
Nevermind, I was on the wrong ver for some reason. Also, it's sad that you cannot use the extension.

**Reply by SpaceMonke135** [reply]
You can use extension. You have to unpack the skiovox webpage file. Then you just go to the webstore and download. If your like me, some of the internet I connect to says, "Download is interrupted" or smth like that. Then if I connect to another different internet, I could download stuff.

**Reply by SpaceMonke135** [reply]
Idk, Skiovox is kinda weird, but it's the best exploit for school chromebook.

**Reply by synapt0** [reply]
> You can use extension. You have to unpack the skiovox webpage file. Then you just go to the webstore and download. 

How does one do this? I'm not sure I understand. (I'm on ver 124)

**Reply by SpaceMonke135** [reply]
Sorry for the late reply, can I see if your doing skiovox on version 124 or 120. Or you computer just updated.
